File: app/src/main/java/com/revosleap/bxplayer/AppUtils/RecyclerView/ViewHolder/TrackViewHolder.java
package com.revosleap.bxplayer.AppUtils.RecyclerView.ViewHolder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.revosleap.bxplayer.R;

public class TrackViewHolder extends RecyclerView.ViewHolder {
   public ImageView trackImage;
   public TextView title,artist;
    public TrackViewHolder(View itemView) {
        super(itemView);
        trackImage= itemView.findViewById(R.id.imageView2);
        title= itemView.findViewById(R.id.textViewTitleTrack);
        artist= itemView.findViewById(R.id.textViewArtistTrack);

    }
}

File: app/src/main/java/com/revosleap/bxplayer/ui/viewholders/TrackViewHolder.kt
package com.revosleap.bxplayer.ui.viewholders

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import android.widget.TextView

import com.revosleap.bxplayer.R

class TrackViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    private var trackImage: ImageView = itemView.findViewById(R.id.imageView2)
    var title: TextView
    var artist: TextView

    init {
        title = itemView.findViewById(R.id.textViewTitleTrack)
        artist = itemView.findViewById(R.id.textViewArtistTrack)

    }
}
